packages/payload/src/uploads/generateFileData.ts

```diff
@@ -25,7 +25,7 @@ type Args<T> = {
   collection: Collection
   config: SanitizedConfig
   data: T
-  operation: 'create' | 'update'
+  operation: 'create' | 'duplicate' | 'update'
```

Comment:
Instead of adding duplicate to the type here can't you just pass `overwriteExistingFiles: true` in the call made from the duplicate operation?